test/src/test/java/hudson/cli/DisconnectNodeCommandTest.java

```diff
@@ -211,27 +176,16 @@ public void disconnectNodeManyShouldSucceedWithCause() throws Exception {
                 .invokeWithArgs(args.toArray(String[]::new));
         assertThat(result, succeededSilently());
         for (var agent : agents) {
-            var computer = agent.toComputer();
-            assertThat(computer.isOffline(), equalTo(true));
-            var cause = computer.getOfflineCause();
-            if (cause instanceof OfflineCause.ByCLI cliCause) {
-                assertThat(cliCause.message, equalTo("aCause"));
-            } else {
-                assertThat("seen occasionally in CI", cause, instanceOf(OfflineCause.ChannelTermination.class));
-            }
+            assertOffline(agent, "aCause");
         }
     }
 
     @Test
     public void disconnectNodeManyShouldFailIfANodeDoesNotExist() throws Exception {
```

Comment:
No flakes seen yet, but since the code is similar to other flaky tests, I'm patching it by reusing `assertOffline`.